framework/widgets/BaseListView.php

```diff
@@ -69,7 +69,9 @@
      */
     public $summaryOptions = ['class' => 'summary'];
     /**
-     * @var boolean whether to show the list view if [[dataProvider]] returns no data.
+     * @var boolean whether to show an empty list view if [[dataProvider]] returns no data.
+     * The default value is false which displays an element according to the `emptyText`
+     * and `emptyTextOptions` properties.
```

Comment:
I agree with that this is confusion. With a `false` for showOnEmpty something is actually "shown". Can we fix this problem in this PR  by renaming the $property (for 2.1 and future releases)